pypit/arcyextract.pyx

```diff
@@ -730,38 +611,37 @@ def extract_mean(np.ndarray[DTYPE_t, ndim=2] frame not None,
         totflux = 0.0
         j=pixll
         donesome=0
-        if dispaxis == 0:
-            # yloc = slf._pixlocn[:,0,0]
-            # xloc = slf._pixlocn[0,:,1]
-            # ysiz = slf._pixlocn[:,0,2]
-            # xsiz = slf._pixlocn[0,:,3]
+        # yloc = slf._pixlocn[:,0,0]
+        # xloc = slf._pixlocn[0,:,1]
+        # ysiz = slf._pixlocn[:,0,2]
+        # xsiz = slf._pixlocn[0,:,3]
+        while True:
```

Comment:
There shouldn't be. `j` is incremented by 1 on line 735, and there's a break statement once j reaches a certain value, on line 620.